src/http-request.ts

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-export interface StackdriverHttpRequest {
+export interface CloudLoggingHttpRequest {
```

Comment:
Would this count as an API-breaking change?

Reply:
I was wondering the same. It is not, as this type is only used internally and not re-exported to users by `index.ts`, and was never exposed in the [client docs here](https://googleapis.dev/nodejs/logging/latest/index.html).